android/app/build.gradle

```diff
@@ -223,6 +223,7 @@ repositories {
 
 dependencies {
     compile project(':react-native-splash-screen')
+    compile project(':react-native-linear-gradient')
```

Comment:
This should be `implementation` instead of `compile`. We can also fix it for the splash screen dependency above.

Android used to use `compile` but switched to `implementation` with one of the last build tools updates. React Native is not up-to-date yet and generates the old style.